eth_test_parser/src/deserialize.rs

```diff
@@ -138,10 +138,27 @@ pub(crate) struct PreAccount {
     pub(crate) storage: HashMap<U256, U256>,
 }
 
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct AccessList {
+    pub(crate) address: Address,
+    pub(crate) storage_keys: Vec<U256>,
+}
+
+#[serde_as]
+#[derive(Deserialize, Debug)]
+/// This is a wrapper around a `Vec<AccessList>` that is used to deserialize a
+/// `null` into an empty vec.
```

Comment:
Weird that some tests use `null` when they could just write `[]` 🤔 but this makes sense 👍